modules/data.land/R/extract_veg.R

```diff
@@ -22,6 +22,41 @@ extract_veg <- function(new_site, start_date, end_date,
                         source, gridres, format_name = NULL, 
                         machine_host, dbparms, outfolder, overwrite = FALSE, input_veg = input_veg, ...){
   #code taken from https://stackoverflow.com/questions/14183766/match-fun-provide-error-with-functions-defined-inside-functions
+   # fget <- function(name, env = parent.frame()) {
+   #    if (identical(env, emptyenv())) {
+   #       stop("Could not find function called ", name, call. = FALSE)
+   #    }
+   #    
+   #    if (exists(name, env, inherits = FALSE) && is.function(env[[name]])) {
+   #       env[[name]]
+   #    } else {
+   #       fget(name, parent.env(env))
+   #    }
+   # }
+ #--------------------------------------------------------------------------------------------------#
```

Comment:
Should this commented-out block be removed?

Reply:
My reasoning to keep it is that if future changes to the extract_veg.R function requires reintroduction of this code chunk then it will be easy to find.